### PR TITLE
Add linting

### PR DESCRIPTION
**Pull request by Minglee01**
@incuna/js 

please review for
https://incuna-pm.atlassian.net/browse/IRD-5

### CODE REVIEW

**Comment by grahamgilchrist** [comment]
Nice!

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+language: node_js
+
+node_js:
+    - '5.0.0'
```
Comment (henrahmagix) [inline .travis.yml:4]: Just `5` will do

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+cache:
+  directories:
+    - $HOME/.npm
+
+sudo: false
+
+before_script:
+    # Start webserver before other tasks so it has time to boot.
+    - 'export DISPLAY=:99.0'
+    - 'sh -e /etc/init.d/xvfb start'
```
Comment (henrahmagix) [inline .travis.yml:18]: You don't need a screen for these tests, so the above 2 lines and comment can be removed

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    - '5.0.0'
+
+notifications:
+    email: false
+
+cache:
+  directories:
+    - $HOME/.npm
+
+sudo: false
```
Comment (henrahmagix) [inline .travis.yml:13]: This is no longer needed: Travis defaults to the faster container setup unless told otherwise

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+node_js:
+    - '5.0.0'
+
+notifications:
+    email: false
+
+cache:
+  directories:
+    - $HOME/.npm
```
Comment (henrahmagix) [inline .travis.yml:11]: Also `$HOME/.cache/bower`

File: alerts.js [hunk 5]
```diff
@@ -6,7 +6,7 @@
 
     alerts.factory('AlertsFactory', [
         '$rootScope',
-        function ($rootScope) {
+        function () {
```
Comment (henrahmagix) [inline alerts.js:9]: Need to remove `'$rootScope'` from the list of dependencies too

File: package.json [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    "grunt-contrib-concat": "1.0.1",
+    "grunt-contrib-uglify": "1.0.1",
+    "grunt-contrib-watch": "1.0.0",
+    "grunt-eslint": "19.0.0",
+    "grunt-force-task": "2.0.0",
+    "grunt-jscs": "3.0.1",
+    "jit-grunt": "0.10.0",
+    "jquery": "3.0.0",
+    "load-grunt-tasks": "3.5.0",
+    "time-grunt": "1.3.0"
```
Comment (henrahmagix) [inline package.json:16]: angular-templates, concat, uglify, watch, force-task, jquery, and jit-grunt aren't used

**Pull request merged**